### PR TITLE
Trophy Fish Total Count

### PR DESCRIPTION
**Pull request by Vixid1**
image

Added trophy fish total count to trophy fish, along with formatted numbers for readability.

### CODE REVIEW

**Comment by romangraef** [comment]
Who is this screenshot from and can we stop them

**Comment by Vixid1** [comment]
> Who is this screenshot from and can we stop them

It’s from DeathStreeks, just needed someone who I know had a lot of trophy fish lol

**Review by romangraef — APPROVED** [review]

**Review by Lulonaut — DISMISSED** [review]

File: src/main/java/io/github/moulberry/notenoughupdates/profileviewer/trophy/TrophyFishPage.java [hunk 1]
```diff
@@ -355,6 +359,8 @@ private List<String> getTooltip(String name, Map<TrophyFish.TrophyFishRarity, In
 		tooltip.add(display(trophyFishRarityIntegerMap, TrophyFish.TrophyFishRarity.GOLD, EnumChatFormatting.GOLD));
 		tooltip.add(display(trophyFishRarityIntegerMap, TrophyFish.TrophyFishRarity.SILVER, EnumChatFormatting.GRAY));
 		tooltip.add(display(trophyFishRarityIntegerMap, TrophyFish.TrophyFishRarity.BRONZE, EnumChatFormatting.DARK_GRAY));
+		tooltip.add(" ");
+		tooltip.add(EnumChatFormatting.GRAY + "Total: " + EnumChatFormatting.GOLD + NUMBER_FORMAT.format((trophyFishList.get(name).getTotal())));
```
Comment (efefury) [inline src/main/java/io/github/moulberry/notenoughupdates/profileviewer/trophy/TrophyFishPage.java:363]: Why are there two `(`?

**Review by efefury — CHANGES_REQUESTED** [review]

File: src/main/java/io/github/moulberry/notenoughupdates/profileviewer/trophy/TrophyFishPage.java (hunk 1 above)
Comment (romangraef) [reply src/main/java/io/github/moulberry/notenoughupdates/profileviewer/trophy/TrophyFishPage.java:363]: all the cool kids use two `(`

Comment (Vixid1) [reply src/main/java/io/github/moulberry/notenoughupdates/profileviewer/trophy/TrophyFishPage.java]: was accidental, fixed in new commit

**Review by hannibal002 — APPROVED** [review]

**Review by NopoTheGamer — APPROVED** [review]

**Review by IRONM00N — APPROVED** [review]

**Pull request merged**